### PR TITLE
Feature/npm scripts info

### PR DESCRIPTION
**Pull request by raphamorim**
Please don't merge it yet

### CODE REVIEW

File: packages/crow-scripts/runners/info.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
+const { spawnPromise, parseOptions } = require('../helpers')
+
+module.exports = ({ infoOptions, extendInfoOptions }) => {
+  const params = parseOptions({
+    overwriteOptions: infoOptions,
+    extendOptions: extendInfoOptions,
+    defaultParams: [
+      'check'
```
Comment (aitherios) [inline packages/crow-scripts/runners/info.js]: npm-scripts-info has the check option?

File: packages/crow-scripts/package.json [hunk 2]
```diff
@@ -6,6 +6,9 @@
   "scripts": {
     "test": "jest --testPathIgnorePatterns __fixtures__"
   },
+  "scripts-info": {
+    "test": "Runs jest tests suit"
```
Comment (aitherios) [inline packages/crow-scripts/package.json]: What do you think about adding a npm task like "scripts-info" to run npm-scripts-info? Maybe we can also use in other packages.

File: packages/crow-scripts/runners/index.js [hunk 3]
```diff
@@ -1,8 +1,9 @@
 module.exports = {
```
Comment (aitherios) [inline packages/crow-scripts/runners/index.js]: Nice!! You ordered it =D

File: packages/crow-scripts/runners/info.js (hunk 1 above)
Comment (raphamorim) [inline packages/crow-scripts/runners/info.js]: updated in 3194947

File: packages/crow-scripts/package.json (hunk 2 above)
Comment (raphamorim) [inline packages/crow-scripts/package.json]: Updated in 498fbac

File: packages/crow-scripts/runners/index.js (hunk 3 above)
Comment (raphamorim) [inline packages/crow-scripts/runners/index.js]: Yey <3

Comment (raphamorim) [inline packages/crow-scripts/runners/index.js]: 305ffb9

**Comment by raphamorim** [comment]
@aitherios can you check the latest change?

**Comment by aitherios** [comment]
sry! merging now

**Pull request merged**